### DISCUSSION TITLE
Can this be turned into tabs

### DISCUSSION THREAD

**Discussion by diomed** [tabler/tabler#1051] [Ideas]
![tabs](https://user-images.githubusercontent.com/72777/154842111-f4d45046-97d0-4b6d-ad2f-6dbc0a2725a3.png)

This would make very nice functional tabs. Can something be done about it?

**Comment by cavasinf** [comment]
You mean like those?

https://preview.tabler.io/cards.html
![image](https://user-images.githubusercontent.com/25293190/221844242-8d17aad5-0cd7-445c-9dc3-158d69e4ee4f.png)

https://getbootstrap.com/docs/5.0/components/navs-tabs/#tabs-with-dropdowns
